Hmm interesting;

```
=== RUN   TestContainerInvalidJSON//containers/foobar/copy
=== PAUSE TestContainerInvalidJSON//containers/foobar/copy
=== CONT  TestContainerInvalidJSON//containers/foobar/copy
=== CONT  TestContainerInvalidJSON//containers/foobar/copy
    container_test.go:28: assertion failed: 404 (res.StatusCode int) != 400 (http.StatusBadRequest int)
    --- FAIL: TestContainerInvalidJSON//containers/foobar/copy (0.00s)

--- a/integration/container/container_test.go
+++ b/integration/container/container_test.go
@@ -13,7 +13,6 @@ func TestContainerInvalidJSON(t *testing.T) {
 	defer setupTest(t)()
 
 	endpoints := []string{
-		"/containers/foobar/copy",
 		"/containers/foobar/exec",
 		"/exec/foobar/start",
 	}

